python/langsmith/client.py

```diff
@@ -742,6 +746,14 @@ def _content_above_size(self, content_length: Optional[int]) -> Optional[str]:
         if size_limit is None:
             return None
         if content_length > size_limit:
+            should_debug_crash_dump = os.getenv("LANGSMITH_DEBUG_CRASH_DUMP") in [
```

Comment:
Let's use `get_env_var` in utils to cache this

Reply:
And define `["1", "true"]` as a tuple and define in outer context

Reply:
Yea I like that!